.gitignore

```diff
@@ -12,3 +12,4 @@ composer.phar
 var/
 composer.lock
 .php_cs.cache
+.idea
```

Comment:
Please do not modify files outside of the scope of a task (specifically avoid using `git add .`).
Moreover this is not the place to add developer's environment-specific configuration. Please use a [global `.gitignore`](https://help.github.com/en/github/using-git/ignoring-files#create-a-global-gitignore) for that.

Reply:
OK, I've reverted to the original source